src/autoTune.ml

```diff
@@ -375,6 +376,15 @@ class octagonVariableVisitor(varMap, globals) = object
         DoChildren
       )
     | Lval ((Var info),_) when not (isGoblintStub info) ->  handle varMap 1 globals (Some (`Right info)) ; SkipChildren
+    | UnOp (LNot, BinOp (op, e1,e2, (TInt _)),_) when isComparison op -> (* TODO: Should be generalized *)
+      handle varMap 5 globals (
+        match extractVar e1, extractVar e2 with
+        | Some a, Some b -> Some (`Left (a,b))
+        | Some a, None
+        | None, Some a -> if isConstant e1 then Some (`Right a) else None
+        | _,_ -> None
+      );
+      DoChildren
```

Comment:
Forgive my ignorance, this is the first time that I look into the autotuner. Is there a particular reason for 

- explicitely handling only one level of `LNot`? Does a recursive match make sense here?
- is there a particular reason to not reuse the `BinOp` case?